src/components/ReminderCtaButton.tsx

```diff
@@ -1,123 +1,110 @@
 import React, { useState } from 'react';
-import { css, ThemeProvider } from '@emotion/react';
-import { neutral, brandAlt, news, body, space } from '@guardian/source-foundations';
+import { css, SerializedStyles, ThemeProvider } from '@emotion/react';
+import { body, space } from '@guardian/source-foundations';
 import { Button } from '@guardian/source-react-components';
 
-import { buildReminderFields, createReminder, ReminderStage } from '../logic/reminders';
-import type { InteractiveButtonStatus } from '../logic/types';
+import {
+    buildReminderFields,
+    createReminder,
+    ReminderStage,
+    ReminderComponent,
+} from '../logic/reminders';
+import type { InteractiveButtonStatus, ColorValueHex } from '../logic/types';
 
 import { FetchEmail } from '../types/dcrTypes';
 import { TrackClick } from '../utils/tracking';
 import { LoadingDots } from './CtaLoadingDotsAnimation';
+import { contributionsTheme, getColors, ReminderButtonColorStyles } from '../styles/colorData';
 
-// Custom theme for Button/LinkButton
-// See also `tertiaryButtonOverrides` below.
-const buttonStyles = {
-    textPrimary: neutral[7],
-    backgroundPrimary: brandAlt[400],
-    backgroundPrimaryHover: brandAlt[300],
-    textSecondary: neutral[7],
-    backgroundSecondary: neutral[93],
-    backgroundSecondaryHover: neutral[86],
-    borderSecondary: neutral[86],
+const defaultButtonColors: ReminderButtonColorStyles = {
+    styleReminderButton: '#121212',
+    styleReminderButtonBackground: '#f6f6f6;',
+    styleReminderButtonHover: '#dcdcdc',
+    styleReminderAnimation: '#707070',
 };
 
-const contributionsTheme = {
-    button: buttonStyles,
-    link: buttonStyles,
-};
-
-const styles = {
-    buttonWrapperStyles: css`
-        margin: ${space[4]}px ${space[2]}px ${space[1]}px 0;
-        display: flex;
-        flex-direction: column;
-        flex-wrap: wrap;
-        justify-content: flex-start;
-        align-items: flex-start;
-        max-width: 50%;
-    `,
-    buttonMargins: css`
-        margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;
-    `,
-    button: css`
-        background-color: ${news[400]};
-        color: ${neutral[100]};
-        &:hover {
-            background-color: ${news[400]};
-        }
-    `,
-    thankYouText: css`
-        ${body.medium({ fontWeight: 'bold' })};
-        color: ${neutral[0]};
-        margin-top: ${space[3]}px;
-    `,
-    remindMeButtonOverrides: css`
-        background-color: transparent !important;
-        color: ${neutral[7]} !important;
-
-        :hover {
-            background-color: ${neutral[86]} !important;
-        }
-    `,
-    smallPrint: css`
-        color: ${neutral[0]};
-        margin-top: ${space[2]}px;
-        ${body.small()};
-    `,
+const getButtonStyles = (
+    userVals: Partial<ReminderButtonColorStyles>,
+    defaults: ReminderButtonColorStyles,
+) => {
+    const styles = getColors(userVals as Partial<ReminderButtonColorStyles>, defaults);
```

Comment:
As in the comment above on `selfServeStyles` I wonder if it'd be better to decouple the two concerns here (merging user provided vals and defaults) and returning CSS. In some cases there won't be user provided styles as they're hardcoded/not configurable. So perhaps the places that need to can do the combining and then call this function with a complete set of colours.